src/constants.ts

```diff
@@ -21,6 +21,16 @@ export enum Channels {
     BOT_COMMANDS = '902990139670814750',
     BIRTHDAY_CHANNEL = '846470774361161808',
     BIRTHDAY_THREAD = '930923893206679573',
+    A32NX_SUPPORT = '785976111875751956',
```

Comment:
If we are adding more here, would it be worth splitting into channels and threads. Or denoting what is a thread by having `_THREAD` like with other entries here?

Reply:
Yeah should look at that. Will work on later